### ISSUE TITLE
activeView always returns "Week"

### ISSUE THREAD

**Issue by daryabsb** [antoniandre/vue-cal-v4#359]
Hello @antoniandre 
your efforts is much appreciated, thank you!
I'm not sure if this problem is the library or my code, but I tried in 5 different ways to make the active-view day, I cant, I even disabled week view. Please help!
![calCapture](https://user-images.githubusercontent.com/58764389/101469101-566e3a80-3955-11eb-8ea2-6362b4010ce1.JPG)

**Comment by antoniandre** [comment]
Thanks for your feedback @daryabsb.
Please share your code and ideally a reproduction link like Codepen, so I can help you!

**Comment by antoniandre** [comment]
Closing the issue due to inactivity. Please reopen with a reproduction link if not solved.

**Comment by daryabsb** [comment]
Sorry to reopen this sir, I tried to create a codepen or something like that, couldn't.
I strongly kindly suggest you create a NUXT.JS version of the calendar, for us who use serverside vue apps, as in my professional opinion, people who need to use calendars, most are the server use people.
At the end, my problem is still there (which I know it is a bug in MY APPLICATION because I tried it with vue, codepen..etc, it worked), and your project is SUPER, THANK YOU!
